authentication/views.py

```diff
@@ -219,11 +222,12 @@ class ClientCreateView(generics.GenericAPIView, BaseUtils):
     Register new client company by client admin
     """
     serializer_class = ClientSerializer
-    renderer_classes = (UserJSONRenderer,)
+    renderer_classes = (ClientJSONRenderer,)
```

Comment:
Why are you changing the renderer?

Reply:
This allows easier and dynamic responses to the data being sent to the user.